examples/06_benchmarks/benchmark_utils.py

```diff
@@ -93,6 +93,8 @@ def recommend_k_als(model, test, train, top_k=DEFAULT_K, remove_seen=True):
         items = train.select(DEFAULT_ITEM_COL).distinct()
         user_item = users.crossJoin(items)
         dfs_pred = model.transform(user_item)
+        spark = start_or_get_spark("ALS PySpark", memory="16g")
+        spark.conf.set("spark.sql.analyzer.failAmbiguousSelfJoin", "false")
```

Comment:
We need these lines of code here, otherwise the notebook raises this error:
ALS:  **AnalysisExceptionError**-- Column itemID#481 are ambiguous. It's probably because you joined several              Datasets together, and some of these Datasets are the same.
I am setting the failAmbigiousSelfJoin to False to fix this.